torch/csrc/distributed/rpc/rpc_agent.cpp

```diff
@@ -4,7 +4,7 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 
-const size_t WorkerId::MAX_NAME_LEN=128;
+constexpr size_t WorkerId::MAX_NAME_LEN;
```

Comment:
I meant using only `constexpr` keyword but inside the header file. Didn't meant `constexpr inline`

Reply:
@mrshenli This is now defined both in the header and here. This one can go.